tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -65,14 +65,14 @@ private static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command, Int
                     IastModule.OnSqlQuery(commandText, integrationId);
                 }
 
-                if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled)
+                if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled && command.CommandType != CommandType.StoredProcedure)
                 {
-                    var propagatedCommand = DatabaseMonitoringPropagator.PropagateSpanData(tracer.Settings.DbmPropagationMode, tracer.DefaultServiceName, scope.Span.Context, integrationId);
+                    var propagatedCommand = DatabaseMonitoringPropagator.PropagateSpanData(tracer.Settings.DbmPropagationMode, tracer.DefaultServiceName, scope.Span.Context, integrationId, out var traceParentInjected);
 
                     if (propagatedCommand != string.Empty)
                     {
                         command.CommandText = $"{propagatedCommand} {commandText}";
-                        tags.DbmDataPropagated = "true";
+                        tags.DbmDataPropagated = traceParentInjected;
```

Comment:
In the RFC I don't see anything that says we should ever inject the value `false` here. AFAICT, we should instead _not set the tag at all_ unless the trace parent is injected. 

In that case, I think you should change `traceParentInjected` to be a `bool`, and do this instead:

```suggestion
                        if (traceParentInjected)
                        {
                            tags.DbmDataPropagated = "true";
                        }
```